### ISSUE TITLE
X to VGM reverse?

### ISSUE THREAD

**Issue by joveaaron** [vgmrips/vgmtools#31]
How could I reverse the process? I want to convert them from VGM to DRO or RAD so I can play them on WinAmp with Adplug. If it's not done yet, are you considering making a reversing tool?

**Comment by vampirefrog** [comment]
Yes, I am considering making a reversing tool, but I don't know when I'll have time for it.

**Comment by vampirefrog** [comment]
There's already a winamp plugin for vgm, so not sure why you'd need to do that.

https://vgmrips.net/forum/viewtopic.php?t=112
